Index: Principios_de_diseno_I/src/Principal/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Principal;\r\n\r\npublic class Main {\r\n\r\n    public static void main(String[] args) {\r\n\t// write your code here\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Principios_de_diseno_I/src/Principal/Main.java b/Principios_de_diseno_I/src/Principal/Main.java
--- a/Principios_de_diseno_I/src/Principal/Main.java	(revision 49085e2fdc1e8551c785c1311a3a118e6c36a0de)
+++ b/Principios_de_diseno_I/src/Principal/Main.java	(date 1637707694379)
@@ -3,6 +3,6 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
     }
 }
